lib/plistLib/main.js

```diff
@@ -127,7 +127,8 @@ define([
             return doc;
         },
         getType: function(value) {
-            if(value instanceof IntObject)
+            if(value instanceof IntObject
+                    || typeof value === 'number' && value === (value|0))
```

Comment:
That relies on the engine shortcutting the OR. It probably will, but there's nothing in the spec that says it has to, so the extra parens harder your code against quirks in other engines.